docs/index.asciidoc

```diff
@@ -173,6 +174,13 @@ Reverse resolve one or more fields.
 
 `resolv` calls will be wrapped in a timeout instance
 
+[id="plugins-{type}s-{plugin}-tags_on_timeout"]
+===== `tags_on_timeout`
```

Comment:
prior art in other plugins is `tag_on_timeout` (see: Grok Filter)

Reply:
I have changed back to `tag_on_timeout` aligning with historical content.